client/src/pages/OrganisationProfile.js

```diff
@@ -395,8 +422,37 @@ const OrganisationProfile = () => {
                   Edit Profile{" "}
                 </Link>
               </DrawerHeader>
+              <DrawerHeader>
+                <div
+                  onClick={handleOrgDelete}
+                  style={{
+                    fontSize: "1.7rem",
+                    color: "#FF5656",
+                  }}
+                >
+                  Delete Organisation
+                </div>
```

Comment:
We also need style `cursor: "pointer"` BUT instead of repeating inline. For the color instead of hardcoding let's also use the `theme.colors.red`. There are two ways - we can import colors to `OrganisationProfile.js` or add a className and update styles in `ProfileComponents.js`.

Here I will suggest a className. 


```suggestion
                <a href="#"
                  role="button"
                  onClick={handleOrgDelete}
                  className="color-danger"
                >
                  Delete Organisation
                </a>
```

Then add to `ProfileComponents.js` line after line 21

```
  a[href].color-danger {
    color: ${colors.red};
  }
```

Reply:
The flash should be gone now and the switch to an href with classname is done